astropy/io/votable/connect.py

```diff
@@ -179,3 +181,98 @@ def write_table_votable(
 io_registry.register_reader("votable", Table, read_table_votable)
 io_registry.register_writer("votable", Table, write_table_votable)
 io_registry.register_identifier("votable", Table, is_votable)
+
+
+# VOTable with embedded/linked Parquet file #
+def write_table_votable_parquet(tab, filename, column_metadata, overwrite):
```

Comment:
Some of these arguments could be optional, and then preferably be mandated to be keyword only.

Also, nitpick: preferably use full words for variables/args/kwargs instead of something abbreviated; and if there is an existing, very similar function already, use the same naming scheme; e.g. the ones in `write_table_votable` could totally be reused.

Reply:
I've added a fix for this